### PR TITLE
Gradle update to version 2.8

### PR DESCRIPTION
**Pull request by vladmm**

### CODE REVIEW

**Comment by mkordas** [comment]
@vladmm, is change from `all` to just `bin` intentional?

**Comment by vladmm** [comment]
Hi @mkordas,
Yes, I'm thinking there's no need for all, is it?

**Comment by leonard84** [comment]
yes, all includes sources

**Comment by vladmm** [comment]
@leonard84 
ok, I will update the patch. I thought the binaries are enough.

**Comment by leonard84** [comment]
The `distributionSha256Sum` is still missing

**Comment by paplorinc** [comment]
should be `2.9` now: https://docs.gradle.org/current/release-notes (an please do use the `-all` version)

**Comment by vladmm** [comment]
@paplorinc I know, I will push a change this evening.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,7 +1,6 @@
-#Mon Sep 14 23:17:03 EEST 2015
+#Sat Oct 31 21:43:43 EET 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionSha256Sum=2ba0aaa11a3e96ec0af31d532d808e1f09cc6dcad0954e637902a1ab544b9e60
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
```
Comment (paplorinc) [inline gradle/wrapper/gradle-wrapper.properties]:
```distributionSha256Sum=496d60c331f8666f99b66d08ff67a880697a7e85a9d9b76ff08814cf97f61a4c
distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip```

**Comment by paplorinc** [comment]
Why aren't simple commits like this merged (or commented)? (or `PR`s updated)
Is `Spock` dead?

**Comment by paplorinc** [comment]
Gradle `2.13` is the latest, what's the point of merging `2.8` 6 months later?

**Comment by szpak** [comment]
@paplorinc This was fixed shortly later in https://github.com/spockframework/spock/commit/b75b7d68c936c83660f0050d6125a7f2a4e95f84.

**Pull request merged**